### ISSUE TITLE
`Polygon<u8>`

### ISSUE THREAD

**Issue by leon0399** [rgeometry/rgeometry#89]
Seems like `Polygon<u8>` is not currently supported, even if you patch and add `fixed_precision!(i8, u8, i16, u16)`, since `PolygonScalar` requires `Neg`

**Comment by lemmih** [comment]
Yeah, `rgeometry` will probably never support unsigned numbers. Too many of the core primitives depend on signed values.
